includes/class-algolia-cli.php

```diff
@@ -68,9 +71,10 @@ public function reindex( $args, $assoc_args ) {
 			WP_CLI::error( 'The configuration for this website does not allow to contact the Algolia API.' );
 		}
 
-		$index_id = isset( $args[0] ) ? $args[0] : null;
-		$clear    = WP_CLI\Utils\get_flag_value( $assoc_args, 'clear' );
-		$all      = WP_CLI\Utils\get_flag_value( $assoc_args, 'all' );
+		$index_id  = isset( $args[0] ) ? $args[0] : null;
+		$clear     = WP_CLI\Utils\get_flag_value( $assoc_args, 'clear' );
+		$all       = WP_CLI\Utils\get_flag_value( $assoc_args, 'all' );
+		$from_page = isset( $assoc_args['from_page'] ) ? intval($assoc_args['from_page']) ? 1;
```

Comment:
Of course. I've never done Wordpress-coding before, so I have to do it blindfolded since the spaces makes my eyes bleed. For obvious reasons there will be typos when coding with blindfolds :)